orderer/common/multichannel/registrar.go

```diff
@@ -370,7 +369,7 @@ func (r *Registrar) SystemChannel() *ChainSupport {
 func (r *Registrar) BroadcastChannelSupport(msg *cb.Envelope) (*cb.ChannelHeader, bool, *ChainSupport, error) {
 	chdr, err := protoutil.ChannelHeader(msg)
 	if err != nil {
-		return nil, false, nil, fmt.Errorf("could not determine channel ID: %s", err)
+		return nil, false, nil, errors.Errorf("could not determine channel ID: %s", err)
```

Comment:
Let's change this one to WithMessage instead. 
```suggestion
		return nil, false, nil, errors.WithMessage(err, "could not determine channel ID")
```